index.html

```diff
@@ -301,20 +324,42 @@ <h3>
     </div>
     <footer>
       <div class="contact">
-        <h2 class="about-me center-text">
-          Contact me
-        </h2>
+        <h2 class="about-me center-text">Contact me</h2>
         <p class="card-para center-text contact-p">
-          If you have an application you are interested in developing, 
-          a feature that you need built or a project that needs coding. I’d love to help with it 
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
         </p>
       </div>
-      <form action="/contact" method="POST">
-        <input type="text" id="username" name="username" Placeholder="Enter your name" value="Yeremias">
-        <input type="email" id="useremail" name="useremail" placeholder="enter your email" value="natajayan14@gmail.com">
-        <textarea name="message" id="message" cols="30" rows="10" placeholder="Write your message here"></textarea>
+      <form action="https://formspree.io/f/xknkbwnn" method="POST">
```

Comment:
- [ ] All the inputs are overflowing and hidden from the left on the screen. Please make sure to give proper styling to fix this and make it look like figma design.

your design:

![Screenshot from 2021-06-08 17-35-50](https://user-images.githubusercontent.com/22434660/121184200-7938c780-c882-11eb-94d1-4f11ab62f529.png)

Reply:
Thanks for the review, I have made most of these changes, however, please I'd appreciate it if you check again, I see the color difference between this screenshot you sent, and what I actually have on the repo.

Please check again